### ISSUE TITLE
API consistency: use loop instead of looping in arcade.play_sound

### ISSUE THREAD

**Issue by pushfoo** [pythonarcade/arcade#1915]
### What should be added/changed?

1. Deprecate or remove the `looping` keyword argument of `arcade.play_sound`
2. Add a `loop` keyword argument to `arcade.play_sound`

### Why?

The following use `loop` as a keyword argument / property name:

*  `pyglet.media.player.Player.play`
* `arcade.Sound.play`

This discrepancy will be annoying for readers using the sound documentation. I discovered it while working on it.

**Comment by Cleptomania** [comment]
I agree with making them consistent, 3.0 is definitely the time to do that. It was different originally because `arcade.play_sound` predates the current pyglet based sound module, and the `play_sound` function was made to be backward's compatible with the old soloud based sound module before we switched to Pyglet.

**Comment by pushfoo** [comment]
tl;dr:
1. So far, I'm fairly convinced we should probably change all discrepancies other than `speed`
2. For the `speed` keyword argument, which sounds less troublesome?

   * Change our playback functions to support both `pitch` and `speed` to specify playback rate of `arcade.Sound`
   * Don't change arcade's use of `speed` despite the pyglet `Player` exposing `pitch`

We have a chance to break compatibility in 3.0 with less consequenes since it aligns with semver.

In pyglet, `pitch` is used for playback control properties and keywords instead of `speed`. From memory, the arguments over `arcade.Sound` went as follows:

* `pitch` should be used because:
   1. arcade should match pyglet
   2. pyglet is a media library, not just a game library
   4. media and music production have an established standard standard laid down by 70s/80s synthesizers

* `speed` should be used becaue:
   1. arcade is a beginner friendly framework
   2. `pitch` is confusing because it implies duration will not be affected
   4. `speed` should be used because it intuitively implies the relationship between pitch and speed is preserved

I argued strongly for the `speed` option, but now I'm second-guessing myself since I've become a strong proponent of API homogeny within and between the libraries. The other option is to leave it as is and just mention that they differ in the doc.

**Comment by Cleptomania** [comment]
In my head, `pitch` makes more sense. Generally, what people want to do is increase the pitch of the sound, however most people just don't realize that implies changing the speed. I think `pitch` is a more accurate term, and also aligns with Pyglet's usage and what you'll see in most media applications.

However, `speed` has worked fine, and I don't know that it's worth breaking the API over, so I'd vote to just leave it with `speed`.

I've went ahead and updated the `loop` argument to be consistent as that wasn't even consistent within our own API, and I think we just leave the `speed` argument for now. If anyone feels strongly that we should be changing that to `pitch`, feel free to re-open this.
